index.js

```diff
@@ -7,7 +6,7 @@ let globalId = 0;
 module.exports = function(source) {
   const id = globalId;
   globalId += 1;
-  const loaderOptions = getOptions(this) || {};
+  const loaderOptions = this.getOptions() || {};
```

Comment:
am I wrong, or is this.getOptions() not defined so loaderOptions would always be {}?

Reply:
https://webpack.js.org/api/loaders/#thisgetoptionsschema

`getOptions` is available in context - it's a method already set on the parent class

check out the code example here:
https://webpack.js.org/contribute/writing-a-loader/#loader-utilities

Reply:
Thanks! I figured I had to be missing something!